package.json

```diff
@@ -77,6 +76,9 @@
     "webpack": "^5.72.1",
     "webpack-cli": "^4.9.2"
   },
+  "optionalDependencies": {
```

Comment:
optionalDependencies are production dependencies (i.e. will be installed by dependents)